pkg/apis/core/v1alpha1/defaults_test.go

```diff
@@ -432,6 +432,11 @@ var _ = Describe("Defaults", func() {
 			Expect(obj.Spec.Kubernetes.KubeAPIServer.Requests.MaxNonMutatingInflight).To(Equal(&maxNonMutatingRequestsInflight))
 			Expect(obj.Spec.Kubernetes.KubeAPIServer.Requests.MaxMutatingInflight).To(Equal(&maxMutatingRequestsInflight))
 		})
+
+		It("should disable anonymous authentication by default", func() {
```

Comment:
If the defaulting is not removed, add a test that verifies the defaulting does not disable explicitly enabled anonymous authentication.

Reply:
Tests provided.